docs/content/posts/2024-03-xx-release-1.19.0.md

```diff
@@ -0,0 +1,501 @@
+* [FLIP-375: Built-in cross-platform powerful java profiler](https://cwiki.apache.org/confluence/x/64lEE)
+
+## New Config Options for Administrator JVM Options
+
+A set of administrator JVM options are available, which prepend the user-set extra JVM options for
+platform-wide JVM tuning.
+
+**More Information**
+* [Documentation](https://nightlies.apache.org/flink/flink-docs-release-1.19/docs/deployment/config/#jvm-and-logging-options)
+* [FLIP-397: Add config options for administrator JVM options](https://cwiki.apache.org/confluence/display/FLINK/FLIP-397%3A+Add+config+options+for+administrator+JVM+options?src=jira)
+
+## Beta Support for Java 21
+
+Apache Flink was made ready to compile and run with Java 21. This feature is still in beta mode.
+Issues should be reported in Flink's bug tracker.
+
+**More Information**
+* [FLINK-33163](https://issues.apache.org/jira/browse/FLINK-33163)
+
+# Checkpoints Improvements
+
+## Using Larger Checkpointing Interval When Source is Processing Backlog
+
+`IsProcessingBacklog` is introduced to indicate whether a record should be processed with low latency
+or high throughput. Connector developers can update the Source implementation to utilize the
+`SplitEnumeratorContext#setIsProcessingBacklog` method to report whether the records are backlog records.
+Users can set the `execution.checkpointing.interval-during-backlog` to use a larger checkpoint interval
+to enhance the throughput while the job is processing backlog if the source is backlog-aware.
+
+**More Information**
+* [FLINK-32514](https://issues.apache.org/jira/browse/FLINK-32514)
+* [FLIP-309: Support using larger checkpointing interval when source is processing backlog](https://cwiki.apache.org/confluence/display/FLINK/FLIP-309%3A+Support+using+larger+checkpointing+interval+when+source+is+processing+backlog)
+
+## CheckpointsCleaner Clean Individual Checkpoint States in Parallel
+
+Now when disposing of no longer needed checkpoints, every state handle/state file will be disposed
+in parallel by the ioExecutor, vastly improving the disposing speed of a single checkpoint (for
+large checkpoints the disposal time can be improved from 10 minutes to < 1 minute) . The old
+behavior can be restored by setting `state.checkpoint.cleaner.parallel-mode` to false.
+
+**More Information**
+* [FLINK-33090](https://issues.apache.org/jira/browse/FLINK-33090)
+
+## Trigger Checkpoints through Command Line Client
+
+The command line interface supports triggering a checkpoint manually. Usage:
+```shell
+./bin/flink checkpoint $JOB_ID [-full]
+```
+By specifying the '-full' option, a full checkpoint is triggered. Otherwise an incremental
+checkpoint is triggered if the job is configured to take incremental ones periodically.
+
+**More Information**
+* [FLINK-6755](https://issues.apache.org/jira/browse/FLINK-6755)
+
+# Connector API Improvements
+
+## New Interfaces to SinkV2 That Are Consistent with SourceV2
```

Comment:
What about `Source` instead of `SourceV2`? We never introduced a v2 API for source

Reply:
@PatrickRen Thanks for the correction! I directly use the title of FLINK-33973, but it's true that we don't have a class like `Sourcev2` in the source code, IIUC, "v2 " source should actually refer to FLIP-27 source.

Reply:
updated